backend/src/main/java/fk/prof/backend/Configuration.java

```diff
@@ -0,0 +1,401 @@
+package fk.prof.backend;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import fk.prof.backend.leader.election.KillBehavior;
+import io.vertx.core.DeploymentOptions;
+import io.vertx.core.VertxOptions;
+import io.vertx.core.http.HttpServerOptions;
+import io.vertx.core.json.JsonObject;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import java.util.Map;
+
+/**
+ * Created by gaurav.ashok on 21/05/17.
+ */
+public class Configuration {
+    @JsonCreator
+    public Configuration(@JsonProperty("ip.address") String ipAddress,
```

Comment:
Is there a better way of doing this mapping from json? ObjectMapper can take a json and map it to a pojo which has annotation applied on class members.
That will be far more readable than having a constructor param